tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+    "compilerOptions": {
+        "target": "es2018",
+        "module": "commonjs",
```

Comment:
I was very conservative on these two options. If necessary we can bump this up to ES2022 for both fields